rbi/core/thread.rbi

```diff
@@ -1140,7 +1140,7 @@ class Thread < Object
   # flag = false # stop thread
   # ```
   sig {params(error: T.class_of(Exception)).returns(T::Boolean)}
-  def self.pending_interrupt?(error); end
+  def self.pending_interrupt?(error = nil); end
```

Comment:
What about `T.unsafe(nil)` instead of plain `nil`?

Reply:
In general I don't really care that much about default values in RBIs for the standard library, they tend to play it pretty fast and loose with typing and also RBI files' default values are not typechecked so it doesn't matter.

Another option is that I believe omitting the `error` argument is akin to passing an error argument of `Exception` (`Thread.pending_interrupt?(Exception)`) so you could use that as the default value if you wanted something which would pass type checks in an alternative universe where RBIs are typechecked.